### DISCUSSION TITLE
The filter cleaning icon (p-table filter-row) is not displayed

### DISCUSSION THREAD

**Discussion by Mazko-Andrei** [primefaces/community#896] [PrimeNG] [answered]
Version primeNG 15.4.1
![Untitled2](https://github.com/primefaces/community/assets/103064672/3ad3c696-17ec-49ba-a391-0a5e2f70eafa)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

This issue is fixed within the 15.4.1 and 17.3.3 versions, could you please try it with the updated versions?
